recipes/erlang/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+./configure --with-ssl="${PREFIX}" --prefix="${PREFIX}"
+make
```

Comment:
Could we please add the following to test this after building?

    make release_tests
    cd release/tests/test_server
    $SRC_DIR/bin/erl -s ts install -s ts smoke_test batch -s init stop